src/VisualStudio/Setup/Roslyn.VisualStudio.Setup.csproj

```diff
@@ -132,6 +132,12 @@
       <ForceIncludeInVSIX>true</ForceIncludeInVSIX>
       <PkgDefEntry>BindingRedirect</PkgDefEntry>
     </ProjectReference>
+    <ProjectReference Include="..\..\Features\LanguageServer\CommonLanguageServerProtocol.Framework\CommonLanguageServerProtocol.Framework.csproj">
```

Comment:
This made MEF happy, but someone should check my work to make sure this is the right way to include it in the package.